daffodil-test/src/test/resources/org/apache/daffodil/section14/sequence_groups/SimilarModelGroupRoot.dfdl.xsd

```diff
@@ -0,0 +1,55 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  Licensed to the Apache Software Foundation (ASF) under one or more
+  contributor license agreements.  See the NOTICE file distributed with
+  this work for additional information regarding copyright ownership.
+  The ASF licenses this file to You under the Apache License, Version 2.0
+  (the "License"); you may not use this file except in compliance with
+  the License.  You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+-->
+
+<!--
+  This schema contents references to two completely different elements
+  (simpleElement1 and simpleElement2), which are exactly the same (including
+  line numbers) except for the root elements and a differentiator. This schema
```

Comment:
differentiator?

Reply:
Yeah, this isn't clear. The specific bug with link16 requires 1) the choice dispatch to be on the same line number with the same contents 2) in separate schemas (and thus a different parent element) 3) and a unique sibling element. Without these three things, the bug would not be triggered. I'll change "differentiator" to "a different sibling element".